runtime/frequency/src/lib.rs

```diff
@@ -1360,6 +1371,9 @@ construct_runtime!(
 		// Allowing accounts to give permission to other accounts to dispatch types of calls from their signed origin
 		Proxy: pallet_proxy = 43,
 
+		// Substrate weights
+		WeightReclaim: cumulus_pallet_weight_reclaim::{Pallet, Storage} = 50,
```

Comment:
nit: why 50?

Reply:
Is there any particular logic to the numbering?
I figured it was a separate group from any of the other "grouped" pallets in the list, so I left a gap before & after. I'm not partial to 50, though. I didn't want to go 68 or 70, figuring we'll want room for expansion in our custom pallets (not that it matters if they're numbered contiguously, but it seemed... cleaner)